src/Features/LanguageServer/Protocol/CustomProtocol/FindUsagesLSPContext.cs

```diff
@@ -216,7 +216,7 @@ public override async ValueTask OnReferenceFoundAsync(SourceReferenceItem refere
 
                 // If we have no source span, our location may be in metadata.
                 var symbol = await SymbolFinder.FindSymbolAtPositionAsync(document, position, cancellationToken).ConfigureAwait(false);
-                if (symbol == null || symbol.Locations == null || symbol.Locations.IsEmpty || !symbol.Locations.First().IsInMetadata)
+                if (symbol == null || symbol.Locations == null || symbol.Locations.IsEmpty)
```

Comment:
Why was this change necessary here? It seems like it's out of sync now with the comment on line 217.

Reply:
I believe the line in 217 is referring to the if-check earlier on line 211. I will move the comment higher and attempt to rephrase it, as I can see how could be confusing.
The change is necessary so that we can use FAR within metadata files. I believe I wrote the original if-statement, but I honestly can't really remember why the last check was included in the first place...